exonum/src/messages2/helpers.rs

```diff
@@ -0,0 +1,43 @@
+use hex::{FromHex, ToHex};
+use serde::{de, ser, Deserialize, Deserializer, Serialize, Serializer};
+
+use std::fmt::Display;
+
+use encoding::Error;
+
+/// Helper trait to define serialization format.
+pub trait BinaryForm: Sized {
+    /// Converts transaction into serialized form.
+    fn serialize(&self) -> Result<Vec<u8>, Error>;
```

Comment:
`serialize` can be confused with the `serde::serialize`, in my opinion `encode` and `decode` or `from_bytes` and `to_bytes` are more convenient.